include/picongpu/particles/Particles.tpp

```diff
@@ -359,7 +359,14 @@ namespace picongpu
             FrameSolver(),
             mapper);
 
-        ParticlesBaseType::template shiftParticles<CORE + BORDER>();
+        shiftBetweenSupercells<CORE + BORDER>();
+    }
+
+    template<typename T_Name, typename T_Flags, typename T_Attributes>
+    template<uint32_t T_area>
+    void Particles<T_Name, T_Flags, T_Attributes>::shiftBetweenSupercells()
+    {
+        ParticlesBaseType::template shiftParticles<T_area>();
```

Comment:
Note: offline discussed with @sbastrakov, this code can not be called for an area containing GUARD. The reason is that shift particles is touching neighbor supercells data in all direction (one neighbor is required)

Reply:
Yes, I will fix it in `dev` tomorrow.